extension/content.js

```diff
@@ -261,8 +261,8 @@ function showRecentlyPushedBranches() {
 }
 
 // Add option for viewing diffs without whitespace changes
-function addDiffViewWithoutWhitespaceOption(type) {
-	if ($('.diff-options-content').length < 1 && $('.btn-group .selected[href*="diff="]').length < 1) {
+function addDiffViewWithoutWhitespaceOption() {
+	if ($('.pr-review-tools .BtnGroup').length < 1 && $('.Details .BtnGroup').length < 1 && $('.pr-review-tools .BtnGroup .tooltipped[href*="diff="]').length < 1) {
```

Comment:
Store `$('.pr-review-tools .BtnGroup')` and `$('.Details .BtnGroup')` into variables, they're used 3 times each.